app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ApplicationRecord
 
   has_many :group_assignments
   has_many :groups, through: :group_assignments
+  has_many :activities, { foreign_key: :owner_id }, -> { order('created_at DESC') }
```

Comment:
`has_many :activities, -> { order(created_at: :desc) }, foreign_key: :owner_id`

If you put the lambda first you don't need to wrap the options.